File: scala-gatling/src/gatling/scala/org/openapitools/client/model/ResponseTimeFilterLocation.scala

package org.openapitools.client.model


case class ResponseTimeFilterLocation (
    _id: String,
    _properties: List[ResponseTimeFilterProperties]
)
object ResponseTimeFilterLocation {
    def toStringBody(var_id: Object, var_properties: Object) =
        s"""
        | {
        | "id":$var_id,"properties":$var_properties
        | }
        """.stripMargin
}

File: fsharp-functions/OpenAPI/src/model/ResponseTimeFilterLocation.fs
namespace OpenAPI.Model

open System
open System.Collections.Generic
open Newtonsoft.Json
open OpenAPI.Model.ResponseTimeFilterProperties

module ResponseTimeFilterLocation = 

  //#region ResponseTimeFilterLocation

  [<CLIMutable>]
  type ResponseTimeFilterLocation = {
    [<JsonProperty(PropertyName = "id")>]
    Id : string;
    [<JsonProperty(PropertyName = "properties")>]
    Properties : ResponseTimeFilterProperties[];
  }
  
  //#endregion
  